esbuild.config.js

```diff
@@ -0,0 +1,44 @@
+const esbuild = require('esbuild');
+const { execSync } = require('child_process');
+const { readFileSync, writeFileSync } = require('fs');
+
+const project = process.env.NX_TASK_TARGET_PROJECT;
+
+esbuild.build({
+  plugins: [
+    {
+      name: 'TypeScriptDeclarations',
+      setup(build) {
+        build.onEnd(result => {
+          if (result.errors.length > 0) return;
+
+          execSync(
+            `tsc --emitDeclarationOnly --project packages/${project}/tsconfig.lib.json --outDir dist`,
```

Comment:
I assume you have tested that its the correct output location :)

Reply:
Yes, I repeatedly ran the `build`s, checked the `dist` folder and checked that I get type information when import the built package.

![image](https://github.com/flowup/quality-metrics-cli/assets/34691111/b0db12ce-6575-4b7a-a65c-9235e0b8207e)

![image](https://github.com/flowup/quality-metrics-cli/assets/34691111/3df5a9c4-6877-4f4c-b27b-83b018a0489b)